sbin/start-worker.sh

```diff
@@ -39,7 +39,7 @@ fi
 # Any changes need to be reflected there.
 CLASS="org.apache.spark.deploy.worker.Worker"
 
-if [[ $# -lt 1 ]] || [[ "$@" = *--help ]] || [[ "$@" = *-h ]]; then
+if [[ $# -lt 1 ]] || [[ "$*" = *--help ]] || [[ "$*" = *-h ]]; then
```

Comment:
SC2199 (error): Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).